torchvision/csrc/io/image/cpu/encode_jpeg.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include <torch/types.h>
```

Comment:
I need to start using a smart editor. I would have just `#include <torch/torch.h>` and bring everything in :-)

Reply:
Unfortunately that's not a smart editor feature but I wish they had that functionality too.